resources/install-letsencrypt-cert.sh

```diff
@@ -25,9 +25,10 @@ read EMAIL
 
 cd /usr/local/sbin
 
-if [ ! -f certbot-auto ] ; then
-  wget https://dl.eff.org/certbot-auto
-  chmod a+x ./certbot-auto
+if [ ! command -v certbot ] ; then
+    sudo add-apt-repository universe
```

Comment:
This won't work on Debian 10, for example. You'll need to detect what distro you're in before deciding to install the package like that. I think the `lsb_release` command can help you detect this.

Reply:
Hey @saghul - Yep, that's why I left it in draft. Unfortunately, I'm unlikely to be able to carry this forward for another week or two. If you (or any other contributor or maintainer) wants to riff on this to make it work more broadly, feel free to do so. 


At present, I don't have the skilzzorz to figure out A) how to take your feedback and translate it into BASH or B) Test this across the target OSes.

If you have specific suggestions for what I could try, or have an automated test suite that provisions a cluster of VM's running the target OSes I'd be down to apply the changes and run your automated checks against it.